src/applications/disability-benefits/all-claims/config/781/index.js

```diff
@@ -180,8 +191,8 @@ export function createFormConfig781a(iterations) {
         ),
         path: `disabilities/ptsd-secondary-incident-description-${index}`,
         depends: isAnswering781aQuestions(index),
-        uiSchema: secondaryIncidentDescription.uiSchema(index),
-        schema: secondaryIncidentDescription.schema(index),
+        uiSchema: secondaryIncidentUnitAssignment.uiSchema(index),
```

Comment:
Did this get swapped with `secondaryIncidentDescription`?

Reply:
Not sure what happened here. Thanks!

Reply:
Looks like a [funky merge](https://github.com/department-of-veterans-affairs/vets-website/pull/9221/commits/30f77316285616950f75d12876007e9bd1d4f2fc).

Reply:
Yeah, totally!

Reply:
Yeah, seriously. Then the new merge with master occurred and changed it even more lol